nrt/data/IEEE14/IEEE14_BasicTestCases/IEEE14_DisconnectLine/reference/outputs/finalStateValues/finalStateValues.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="ISO-8859-1" standalone="no"?>
+<finalStateValuesOutput xmlns="http://www.rte-france.com/dynawo">
+  <finalStateValue model="NETWORK" variable="_BUS____1_TN_Upu_value" value="1.062466"/>
```

Comment:
We might need to add a smart float comparison for those values in the nrt scripts. Can be done in another PR.